packages/cozy-mespapiers-lib/src/utils/input.js

```diff
@@ -25,7 +25,13 @@ export const makeConstraintsOfInput = attrs => {
   const { type = '', required = false, mask } = attrs || {}
 
   const maskLength = mask?.replace(/\s/g, '')?.length
-  const minLength = maskLength ?? attrs?.minLength ?? null
+  const minLength = required
+    ? maskLength ?? attrs?.minLength ?? null
+    : attrs?.minLength != null
+    ? maskLength
+      ? Math.min(attrs.minLength, maskLength)
+      : attrs.minLength
+    : null
```

Comment:
Something like that?
```suggestion
const minLength = attrs?.minLength ?? maskLength ?? null
```

Reply:
Approche changée